docs/guides/validation/validation_actions/how_to_trigger_email_as_a_validation_action.md

```diff
@@ -28,61 +29,56 @@ sender_alias: [optional alias used to send the email (default = sender_login)]
 receiver_emails: [addresses you want to send the email to]  # each address must be separated by commas
 ````
 
-### 2. Update `validation_operators` in `great_expectations.yml`
+### 2. Update `action_list` in your Checkpoint configuration
 
 Open `great_expectations.yml` and add `send_email_on_validation_result` Action to `validation_operators`. Make sure the following section exists in the `great_expectations.yml` file.
 
-````python
-validation_operators:
-    action_list_operator:
-        # To learn how to configure sending emails during evaluation
-        # (and other customizations), (/docs/reference/checkpoints-and-actions/) 
-        class_name: ActionListValidationOperator
-        action_list:
-        #--------------------------------
-        # here is what you will be adding
-        #--------------------------------
-        - name: send_email_on_validation_result # name can be set to any value
-          action:
-            class_name: EmailAction
-            notify_on: all # possible values: "all", "failure", "success"
-            notify_with: # optional list containing the DataDocs sites to include in the notification. Defaults to including links to all configured sites.
-            # You can choose between using SSL encryption, TLS encryption or none of them (not advised)
-            use_tls: False
-            use_ssl: True
-            renderer:
-              module_name: great_expectations.render.renderer.email_renderer
-              class_name: EmailRenderer
-            # put the actual following information in the uncommitted/config_variables.yml file
-            # or pass in as environment variable
-            smtp_address: ${smtp_address}
-            smtp_port: ${smtp_port}
-            sender_login: ${sender_login}
-            sender_password: ${sender_password}
-            sender_alias: ${sender_alias}
-            receiver_emails: ${receiver_emails}  # string containing email addresses separated by commas
+````yaml
+action_list:
+    #--------------------------------
+    # here is what you will be adding
+    #--------------------------------
```

Comment:
I like this, but it is inconsistent with the other PRs on OpenLineage, OpsGenie, and Slack notifications

Reply:
Fixed; removed for consistency.  I think this was just left over from converting this away from the validation  operator version, when there was more above the `action_list` key.